include/grpcpp/impl/codegen/client_context.h

```diff
@@ -200,8 +200,9 @@ class ClientContext {
   ///
   /// \return A multimap of initial metadata key-value pairs from the server.
   const std::multimap<grpc::string_ref, grpc::string_ref>&
-  GetServerInitialMetadata() const {
+  GetServerInitialMetadata() {
```

Comment:
Changing this to be non-const is a public API change.  I'm not sure that's okay.

The only alternative I can see would be to declare the `recv_initial_metadata_` member as `mutable`, although that's pretty ugly.

Reply:
Done